internal/server/migrations.go

```diff
@@ -89,13 +89,13 @@ func (a *API) addResponseRewrites() {
 	addResponseRewrite(a, "get", "/v1/access-keys", "0.12.2", func(newResponse *api.ListResponse[api.AccessKey]) []api.AccessKey {
 		return newResponse.Items
 	})
-	addResponseRewrite(a, "get", "/v1/identities", "0.12.2", func(newResponse *api.ListResponse[api.User]) []api.User {
+	addResponseRewrite(a, "get", "/v1/users", "0.12.2", func(newResponse *api.ListResponse[api.User]) []api.User {
```

Comment:
This is necessary because the path has been redirected from `/v1/identities` to `/v1/users`

Reply:
Yeah, you'll want to leave this file unchanged.

Reply:
I mean this change is necessary _to_ have the correct migration behaviour. Otherwise the redirect modifies the path from `/v1/identities` to `/v1/users` causing it to no longer match this path

Reply:
This change can be omitted once #1943 is merged